### DISCUSSION TITLE
Setup README out of date

### DISCUSSION THREAD

**Discussion by Zeyu-Li** [lowlighter/metrics#823] [General]
This part of the readme is out of date since GitHub updated the settings tab: https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/action.md#2%EF%B8%8F-put-your-github-personal-token-in-repository-secrets

![image](https://user-images.githubusercontent.com/34993025/151379782-e379ef54-de5b-4c34-b067-db0e7553ddef.png)

**Comment by lowlighter** [comment]
I saw it's still changing now (they're still trying to find which icons are the best it seems)
It'll be updated once we're sure they have finally settled for a design 👍

**Reply by lowlighter** [reply]
Tracked in #860
